src/components/tiles-container/TilesContainer.js

```diff
@@ -131,14 +128,12 @@ class TilesContainer extends Component {
             projects.map(feature => (
               <ListItem
                 data={feature}
-                key={feature.attributes.OBJECTID}
+                key={feature.mpms}
```

Comment:
if OBJECTID is still available, switch back to it. Keys have to be unique

Reply:
Will check within mapbox gl feature for unique id, no need to have any AGO jawns weaseling back in.